config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -392,3 +392,297 @@ periodics:
   annotations:
     testgrid-dashboards: sig-scalability-experiments
     testgrid-tab-name: gce-private-cluster-correctness
+# max volumes per pod test cases for ephemeral volumes
+- name: ci-kubernetes-storage-scalability-max-emptydir-vol-per-pod
+  interval: 1h
+  labels:
+  #TODO which labels I need?
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+    preset-e2e-scalability-node: "true"
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20190716-41e2a9e-master
+      args:
+      - --repo=k8s.io/kubernetes=master
+      - --repo=k8s.io/perf-tests=master
+      - --root=/go/src
+      - --timeout=60
+      - --scenario=kubernetes_e2e
+      - --
+      - --check-leaked-resources
+      - --cluster=
+      - --extract=ci/latest
+      - --gcp-node-image=gci
+      - --gcp-nodes=1
+      - --provider=gce
+      - --test=false
+      - --test-cmd=$GOPATH/src/k8s.io/perf-tests/run-e2e.sh
+      - --test-cmd-args=cluster-loader2
+      - --test-cmd-args=--nodes=1
+      - --test-cmd-args=--provider=gce
+      - --test-cmd-args=--report-dir=/workspace/_artifacts
+      - --test-cmd-args=--testconfig=testing/experimental/storage/pod-startup/ephemeral-volumes/1_node/config.yaml
```

Comment:
nit: I think it would be a bit more intuitive to add one more override for emptydir (even though it's the default) so that we will know what exactly is happening here (and for consistency with other definitions).

Reply:
I agree, explicit is better. I also will do the same thing for default test case(that's max volume per pod), this will be another PR for perf-tests.

Reply:
https://github.com/kubernetes/perf-tests/pull/653